modules/460-log-shipper/crds/cluster-log-destination.yaml

```diff
@@ -315,6 +323,54 @@ spec:
                           type: boolean
                           default: true
                           description: Validate the TLS certificate of the remote host.
+                vector:
+                  type: object
+                  required:
+                    - endpoint
+                  properties:
+                    endpoint:
+                      type: string
+                      description: The base URL of the Vector instance. API v2 must be used for communication between instances.
```

Comment:
Description talks about base URL but pattern requires Host:Port endpoint. It could be confusable